src/main/resources/assets/eln/lang/zh_CN.lang

```diff
@@ -1,234 +1,234 @@
 electrical_furnace.name=电炉
-voltage-to-redstone_converter.name=信号-红石转换器
-small_50v_incandescent_light_bulb.name=小型50V白炽灯泡
-800v_defence_turret.name=800V步哨枪
-200v_iron_heating_corp.name=200V Iron Heating Corp
-200v_turbine.name=200V涡轮机
-industrial_data_logger.name=工业数据显示器
+voltage-to-redstone_converter.name=电压转红石器
+small_50v_incandescent_light_bulb.name=小型50V白炽灯灯泡
+800v_defence_turret.name=800V防卫炮塔
+200v_iron_heating_corp.name=200V铁加热组
+200v_turbine.name=200V汽轮机
+industrial_data_logger.name=工业数据记录器
 silicon_plate.name=硅板
 advanced_magnet.name=高级磁铁
-long_suspended_lamp_socket.name=长链灯座
+long_suspended_lamp_socket.name=长悬挂式电灯插座
 signal_switch.name=信号开关
-signal_20h_inductor.name=Signal 20H inductor
+signal_20h_inductor.name=信号20H感应器
 200v_compressor.name=200V压缩机
 50v_compressor.name=50V压缩机
 machine_block.name=机械方块
-50v_plate_machine.name=50V轧板机
-voltage_probe.name=电压探针
-electrical_weather_sensor.name=气候传感器
+50v_plate_machine.name=50V制板机
+voltage_probe.name=电压测量器
+electrical_weather_sensor.name=电子天气传感器
 signal_diode.name=信号二极管
 green_brush.name=绿色刷子
-200v_condensator.name=200V Condensator charged at 0%
-cost_oriented_battery.name=普通电池（50%电量）
+200v_condensator.name=200V电容 储量:0%
+cost_oriented_battery.name=电池 储量:50%
 brown_brush.name=棕色刷子
-cheap_ferromagnetic_core.name=廉价铁磁核心
-portable_electrical_mining_drill.name=便携式采矿电钻
-small_50v_iron_heating_corp.name=小型50V铁质导热线
-monster_filter.name=怪物过滤镜
-electrical_light_sensor.name=电子光敏感器
+cheap_ferromagnetic_core.name=低级铁磁核心
+portable_electrical_mining_drill.name=便携电子挖矿钻头
+small_50v_iron_heating_corp.name=小型50V铁加热组
+monster_filter.name=怪物过滤器
+electrical_light_sensor.name=电子光传感器
 magenta_brush.name=品红色刷子
 simple_lamp_socket.name=简易灯座
-high_power_transmitter_antenna.name=大功率发射天线
-signal_processor.name=信号处理机
+high_power_transmitter_antenna.name=高功率发射天线
+signal_processor.name=信号处理器
 orange_brush.name=橙色刷子
-pink_brush.name=粉红色刷子
-wireless_signal_transmitter.name=无线信号发射器
+pink_brush.name=粉色刷子
+wireless_signal_transmitter.name=无线信号发生器
 wireless_button.name=无线按钮
-very_high_voltage_switch.name=超高电压开关
-nuclear_alarm.name=警报器
-50v_turbine.name=50V涡轮机
+very_high_voltage_switch.name=超高压开关
+nuclear_alarm.name=核能报警器
+50v_turbine.name=50V汽轮机
 gray_brush.name=灰色刷子
-200v_incandescent_light_bulb.name=200V白炽灯泡
+200v_incandescent_light_bulb.name=200V白炽灯灯泡
 white_brush.name=白色刷子
-tutorial_sign.name=教程标志
+tutorial_sign.name=教程标记
 lead_dust.name=铅粉
 10a_diode.name=10A二极管
-advanced_energy_meter.name=高级能量仪表
+advanced_energy_meter.name=高级能量计
 small_active_thermal_dissipator.name=小型主动散热器
-cheap_electrical_drill.name=廉价电钻
-small_50v_economic_light_bulb.name=小型50V节能灯泡
-capacity_oriented_battery.name=容量升级电池（50%电量）
-electrical_probe_chip.name=电力探测芯片
-thermometer.name=温度表
-on/off_regulator_1%.name=On/OFF Regulator 1%
+cheap_electrical_drill.name=低级电子钻头
+small_50v_economic_light_bulb.name=小型50V经济灯泡
+capacity_oriented_battery.name=容量强化电池 储能:50%
+electrical_probe_chip.name=电子测量芯片
+thermometer.name=温度计
+on/off_regulator_1%.name=On/OFF调节器 1%
```

Comment:
In such case "On" should be "ON" (i.e. all capitalised). The same as previous entries.